india_compliance/gst_india/api_classes/nic/auth.py

```
@@ -0,0 +1,181 @@
+    def _decrypt_session_key(self, response):
+        # For Auth API
+        values = {}
+        key = self.client.getkey("AuthToken")
+        if response.get(key):
+            self.auth_token = response[key]
+            values["auth_token"] = response[key]
+
+        key = self.client.getkey("Sek")
+        if response.get(key):
+            self.session_key = aes_decrypt_data(
+                response[key], base64.b64decode(self.app_key.encode())
```

Pylint (3.3.7)

[error] Instance of 'StandardAuth' has no 'app_key' member

(E1101)